NDB.Covid19/NDB.Covid19/Locales/nb.json

```diff
@@ -97,7 +97,7 @@
   "SETTINGS_PAGE_5_LINK": "www.helsenorge.no/smittestopp",
   "FORCE_UPDATE_MESSAGE": "Det finnes en nyere versjon av Smittestopp. <br><br> Oppdater appen for å fortsette.",
   "FORCE_UPDATE_BUTTON_GOOGLE_ANDROID": "Gå til Google Play Butikk",
-  "FORCE_UPDATE_BUTTON_HUAWEI_ANDROID": "Gå til AppGallery",
+  "FORCE_UPDATE_BUTTON_HUAWEI_ANDROID": "Gå til App Gallery",
```

Comment:
Om jeg ser på Huaweis side https://huaweimobileservices.com/appgallery/  , så er teksten der "Make AppGallery your Discovery Centre",  "AppGallery News" og meny tittelen der er også "AppGallery".  Ser at det finnes annt to-ords meny del der, så det ser ut som korrekt er at det skal hete AppGallery i ett ord.  Men, når det over står "Google Play Butikk", burde det da ikke her stå "Huawei AppGallery" ?  @pavshr ?